app/controllers/user.js

```diff
@@ -54,3 +82,29 @@ exports.signUp = ({ user }, res, next) =>
         );
     })
     .catch(next);
+
+exports.usersPerPage = 2;
+
+exports.listUsers = (req, res, next) => {
+  try {
+    const decoded = jwt.decode(req.headers.token);
+  } catch (e) {
+    next(errors.badRequest(errorMsgs.invalidToken));
+    return;
+  }
+  const where = { limit: exports.usersPerPage, offset: 0 };
+  if (req.headers.page) where.offset = req.headers.page * exports.usersPerPage;
```

Comment:
Page and limit in query (url, for example endpoint?page=0&limit=0). Use a default value for page and limit